hardware_control:
  #rc_car:
    #steer_max_angle: 45
    #throttle: 1515
  platform:   # 1/4 scale platform
    steer_max_angle: 26
    throttle: 7
  # control_error *= factor 
  factor:
    yaw: 0.4
    lateral: 0.8
